dbfit-java/sqlserver/src/main/java/dbfit/environment/SqlServerEnvironment.java

```diff
@@ -67,9 +68,16 @@ protected String parseCommandText(String commandText) {
 
     public Map<String, DbParameterAccessor> getAllColumns(String tableOrViewName)
             throws SQLException {
+
+        String objectDatabaseName = "";
+        String[] objnameParts = tableOrViewName.split("\\.");
+        if (objnameParts.length == 3) {
+            objectDatabaseName = objnameParts[0] + ".";
```

Comment:
`objectDatabaseNameDot`?! Or I can just use a ternary operator (`+ objectDatabaseName + (objectDatabaseName == "") ? "" : ".")` to include the `.` separator.

Any views?